libs/cardano-ledger-core/src/Cardano/Ledger/TerseTools.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Cardano.Ledger.TerseTools where
```

Comment:
maybe this could live in `libs/cardano-ledger-pretty` instead?

Reply:
The problem with that is, that it can't be used when debugging Shelley code, because of mutual recursion problems.  The point of TerseTools is to make "terse" representations of the differences between two objects/ Not to pretty print them.

Reply:
ah. how about `eras/shelley/impl/src/Cardano/Ledger/Terse.hs`?

Reply:
So moved.